src/DynamoRevit/ViewModel/DynamoRevitViewModel.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+
+using Autodesk.Revit.DB;
+
+using Dynamo.Applications.Models;
+using Dynamo.Applications.Properties;
+using Dynamo.Interfaces;
+using Dynamo.Models;
+using Dynamo.ViewModels;
+using Dynamo.Wpf.ViewModels.Core;
+
+using RevitServices.Persistence;
+
+namespace Dynamo.Applications.ViewModel
+{
+    public class DynamoRevitViewModel : DynamoViewModel
+    {
+        private DynamoRevitViewModel(StartConfiguration startConfiguration) :
+            base(startConfiguration)
+        {
+            var model = (RevitDynamoModel)Model;
```

Comment:
Yeah. I started to refactor ViewModel to ViewModel<T> and then stopped, because although that is the right thing to do (so you don't have to do this kind of cast), it's a big job.